src/ComplexType.php

```diff
@@ -117,7 +117,11 @@ protected function generateClass()
                 $getterCode = '  if ($this->' . $name . ' == null) {' . PHP_EOL
                     . '    return null;' . PHP_EOL
                     . '  } else {' . PHP_EOL
-                    . '    return \DateTime::createFromFormat(\DateTime::ATOM, $this->' . $name . ');' . PHP_EOL
+                    . '    try {' . PHP_EOL
+                    . '      return new \DateTime($this->' . $name . ');' . PHP_EOL
+                    . '    } catch (\Exception $e) {' . PHP_EOL
+                    . '      return null;' . PHP_EOL
```

Comment:
@kasperg Yes, you're right but I've decided to change it to null because the getter returns an object. The object is instance of \DateTime or can be null. I don't feel free when I'm expecting an object from API and I get a false value. If we need to know the conversion fails I can replace a "return null" to "return false" and you'll be good with this code?